playbooks/group_vars/aptly_servers.yaml

```diff
@@ -0,0 +1,14 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+aptly__root_dir: /srv/debian
+aptly__repo_group: debrepo
+
+aptly__repo_dists:
+  - bullseye
+  - bookworm
+  - unstable
+
+# TODO(dato): make `aptly__fspub_endpoints` into a map or list.
+# FIXME(dato): we should probably have no default for the endpoint name?
+aptly__fspub_endpoint_name: tina.pm
```

Comment:
The name is really not very important, but better default would be `debian.{{ domain_name }}` or just `filesystem`

Reply:
Will do